templates/practice/documentation.html

```diff
@@ -0,0 +1,246 @@
+
+{% block meta_description %}At Canonical, we aim to set a standard of excellence in technical documentation, by treating it as a professional discipline within our engineering practice.{% endblock %}
+
+{% block meta_image %}https://assets.ubuntu.com/v1/253da317-image-document-ubuntudocs.svg{% endblock %}
+
+{% block content %}
+<section class="p-strip--suru-topped is-bordered">
+  <div class="row">
+    <div class="col-7">
+      <h1>Documentation at Canonical</h1>
+      <p>At Canonical, we have embarked on a comprehensive, long-term project to transform documentation.</p>
+      <p>Our aim is to create and maintain documentation product and practice that will represent a standard of excellence. We want documentation to be the best it possibly can be.</p>
+    </div>
+    <div class="col-5 u-vertically-center u-align--center u-hide--small u-hide--medium">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/253da317-image-document-ubuntudocs.svg",
+        alt="",
+        width="214",
+        height="248",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+<section class="p-strip is-deep">
+  <div class="row" id="documentation">
+    <aside class="col-3">
+      <div class="p-side-navigation is-sticky" id="drawer" style="top: 4.5rem; z-index: 39;">
+        <a href="#drawer" class="p-side-navigation__toggle js-drawer-toggle" aria-controls="drawer">
+          Toggle table of contents
+        </a>
+        <div class="p-side-navigation__overlay js-drawer-toggle" aria-controls="drawer"></div>
+        <nav class="p-side-navigation__drawer" aria-label="Table of content" style="top: 56px;">
+          <div class="p-side-navigation__drawer-header">
+            <button class="p-side-navigation__toggle--in-drawer js-drawer-toggle" aria-controls="drawer">
+              Toggle table of contents
+            </button>
+          </div>
+          <ul class="p-side-navigation__list">
+            <li class="p-side-navigation__item">
+              <a class="p-side-navigation__link" href="#documentation-introduction">Introduction</a>
+            </li>
+            <li class="p-side-navigation__item">
+              <a class="p-side-navigation__link" href="#documentation-pillars">Four pillars</a>
+            </li>
+            <li class="p-side-navigation__item">
+              <a class="p-side-navigation__link" href="#documentation-careers">Careers</a>
+            </li>
+            <li class="p-side-navigation__item">
+              <a class="p-side-navigation__link" href="#documentation-milestones">Milestones</a>
+            </li>
+            <li class="p-side-navigation__item">
+              <a class="p-side-navigation__link" href="#documentation-readmore">Read more</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </aside>
+    <main class="col-9">
+      <div id="documentation-introduction" style="scroll-margin-top: 4.5rem;">
```

Comment:
Could we just use "introduction". This reduces the duplication of documentation in the URL.